### DISCUSSION TITLE
Pair Ext achievement

### DISCUSSION THREAD

**Discussion by VsevolodDiachun** [community/community#116655] [Discover: GitHub Best Practices]
Do you know how to get Pair Ext achievement in GitHub one more time???

**Comment by KaplunMaxym** [comment]
You should use GitHub Desktop in order to add co-authors to your commit
<img width="249" alt="Знімок екрана 2024-04-03 о 16 51 21" src="https://github.com/community/community/assets/61882640/3241f924-748c-4e3c-90ab-1dd82feaa6a2">

**Comment by queenofcorgis** [comment]
Hi @VsevolodDiachun  

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
